experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts

```diff
@@ -60,33 +36,59 @@ export async function performFuzzActions(
  * See the "Fuzz - Targeted" test suite for tests that validate more specific code paths or invariants.
  */
 describe("Fuzz - Top-Level", () => {
-	const random = makeRandom(0);
 	const runsPerBatch = 20;
 	const opsPerRun = 20;
 	const editGeneratorOpWeights: Partial<EditGeneratorOpWeights> = {
 		setPayload: 1,
 	};
+	const generatorFactory = () => takeAsync(opsPerRun, makeOpGenerator(editGeneratorOpWeights));
 	/**
 	 * This test suite is meant exercise all public APIs of SharedTree together, as well as all service-oriented
 	 * operations (such as summarization and stashed ops).
 	 */
 	describe("Everything", () => {
-		runFuzzBatch(
-			makeOpGenerator,
-			performFuzzActions,
-			opsPerRun,
-			runsPerBatch,
-			random,
-			editGeneratorOpWeights,
-		);
+		const model: DDSFuzzModel<
+			SharedTreeTestFactory,
+			Operation,
+			DDSFuzzTestState<SharedTreeTestFactory>
+		> = {
+			workloadName: "SharedTree",
+			factory: new SharedTreeTestFactory(onCreate),
+			generatorFactory,
+			reducer: fuzzReducer,
+			validateConsistency: validateTreeConsistency,
+		};
+		const options = {
+			...baseOptions,
+			defaultTestCount: runsPerBatch,
+		};
+		createDDSFuzzSuite(model, options);
 	});
 });
 
 describe.skip("Re-run form ops saved on file", () => {
 	// For using saved operations set the value of the runSeed used to saved the ops in the file.
 	const runSeed = 0;
 	const filepath = path.join(__dirname, `fuzz-tests-saved-ops/ops_with_seed_${runSeed}`);
+	const generatorFactory = () => makeOpGenerator();
```

Comment:
this entire skipped describe block can be deleted. `createDDSFuzzSuite` takes in a `replay` option which you can use to get this functionality. So you'll see a lot of usages to leave that option source-controlled but commented out with some number, so that's more obvious to people debugging the test.

Reply:
removed describe block